ui/webui/resources/polymer_overriding.js

```diff
@@ -73,6 +88,18 @@ export function RegisterPolymerComponentBehaviors(behaviorsMap) {
   Object.assign(allBehaviorsMap, behaviorsMap)
 }
 
+export function RegisterPolymerComponentProperties(propertiesMap) {
+  if (debug) {
+    console.error('RegisterPolymerComponentProperties', ...Object.keys(propertiesMap))
```

Comment:
IMO would be better to have console.debug (or console.log) here so that we don't have things that look like errors in the long list of log entries that occur when `debug` flag is on

Reply:
@petemill - updated